server/sql/postgraphql/postgraphql.sql

```diff
@@ -0,0 +1,18 @@
+CREATE SCHEMA IF NOT EXISTS postgraphql;
+
+DO $$
+	BEGIN
+		PERFORM * FROM pg_catalog.pg_matviews WHERE matviewname = 'organization' AND schemaname = 'postgraphql';
+		IF NOT FOUND THEN
+			CREATE MATERIALIZED VIEW postgraphql.organization AS
+				SELECT
+					organization.*,
+					now() AS last_refresh,
+					(SELECT max(last_update) FROM incoming.project) AS last_update
+				FROM report.organization;
+			CREATE UNIQUE INDEX postgraphql_organization_index ON postgraphql.organization(orgname);
+		ELSE
```

Comment:
does this give all coderbunker clients a unique ID?

Reply:
no, the constraint on name being unique is in model.organization:

https://github.com/coderbunker/timesheet-backend/blob/master/server/sql/model/model.sql#L25

This is an index on that unique name that makes retrieval an O(1) operation instead of a sequential search O(n). Also a requirement for MATERIALIZED VIEW (cached data) refresh.